docs/en/query_language/functions/string_search_functions.md

```diff
@@ -2,12 +2,89 @@
 
 The search is case-sensitive by default in all these functions. There are separate variants for case insensitive search.
 
-## position(haystack, needle), locate(haystack, needle)
+## position(haystack, needle) {#position}
 
-Search for the substring `needle` in the string `haystack`.
-Returns the position (in bytes) of the found substring, starting from 1, or returns 0 if the substring was not found.
+Returns the position (in bytes) of the found substring `needle` in the string `haystack`, starting from 1. Returns 0 if the substring was not found.
 
-For a case-insensitive search, use the function `positionCaseInsensitive`.
+Works under the assumption that the string contains a set of bytes representing a single-byte encoded text. If this assumption is not met, it returns some result.
+
+For a case-insensitive search, use the function [positionCaseInsensitive](#positioncaseinsensitive).
```

Comment:
Why do you mention only CaseInsensitive version, but not UTF8?

Reply:
It hasn't been described yet

Reply:
why not?

Reply:
already described, see files